node/follower_node_test.go

```diff
@@ -157,7 +157,7 @@ func TestSyncRoundWithRemake(t *testing.T) {
 	partitiontest.PartitionTest(t)
 	t.Parallel()
 
-	maxAcctLookback := uint64(100)
+	maxAcctLookback := uint64(10)
```

Comment:
please fix a comment on line 187
`	// * cfg.MaxAcctLookback = 100 (instead of 4)` -> `	// * cfg.MaxAcctLookback = 10 (instead of 4)`

Reply:
043fb95cad9542834077b395a56489e7ee844495